### PR TITLE
TINY-12773: Add unit tests for AutoResizingTextareaUtils helpers

### PR DESCRIPTION
**Pull request by hamza0867**
Related Ticket: TINY-12773

Description of Changes:

- Added unit tests for `computeMaxRows` and `computeMinRows` in `AutoResizingTextareaUtils`, covering `rows` and `px` units, clamping behaviour, flooring/ceiling of fractional results, and edge cases like zero and negative values
- Added browser tests for `isVisible`, `computeSingleRowHeight`, and `resizeTextarea`, covering visibility detection (including fixed-positioned elements), row height measurement fallbacks, state restoration after measuring, and row clamping to min/max bounds
- Updated `CLAUDE.md` command references from `yarn` to `bun run`

Pre-checks:

- [x] Changelog entry added
- [x] Tests have been added (if applicable)
- [x] Branch prefixed with `feature/`, `hotfix/` or `spike/`

Review:

- [x] Milestone set
- [x] Docs ticket created (if applicable)

GitHub issues (if applicable):

## Summary by CodeRabbit

* **Documentation**
  * Updated repository command documentation for development setup.

* **Tests**
  * Added comprehensive test coverage for auto-resizing textarea functionality.

* **Chores**
  * Updated configuration to optimize dependency handling during testing.

### CODE REVIEW

**Comment by hamza0867** [comment]
* **#11096** <a href="https://app.graphite.com/github/pr/tinymce/tinymce/11096?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.com/github/pr/tinymce/tinymce/11096?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* **#11095** <a href="https://app.graphite.com/github/pr/tinymce/tinymce/11095?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`


<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

**Review by TheSpyder — APPROVED** [review]
Why are these a separate PR? I don't think that's how stacked commits are supposed to work.

File: modules/oxide-components/vitest.config.ts [hunk 1]
```diff
@@ -11,6 +11,17 @@ export default defineConfig({
...
           environment: 'node',
+          server: {
+            deps: {
+              // @ephox/dispute ships "type": "module" with extensionless relative imports,
+              // which Node's strict ESM loader rejects. Inlining routes @ephox packages
+              // through Vite's resolver, which tolerates the missing extension. The whole
+              // chain (sugar → katamari → dispute) must be inlined: if a parent stays
+              // externalized, Node loads it directly and its `import '@ephox/dispute'`
+              // never reaches Vite.
+              inline: [ '@ephox' ]
```
Comment (spocke) [inline modules/oxide-components/vitest.config.ts:22]: I hope that the TS team comes to their senses some day and adds a compiler option to enforce this. I've seen scripts and hacks for this for years.

**Review by spocke — APPROVED** [review]

**Comment by hamza0867** [comment]
### Merge activity

* **May 8, 8:47 AM UTC**: A user started a stack merge that includes this pull request via [Graphite](https://app.graphite.com/github/pr/tinymce/tinymce/11096).
* **May 8, 8:49 AM UTC**: [Graphite](https://app.graphite.com/github/pr/tinymce/tinymce/11096) rebased this pull request as part of a merge.
* **May 8, 9:07 AM UTC**: @hamza0867 merged this pull request with [Graphite](https://app.graphite.com/github/pr/tinymce/tinymce/11096).
